modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -62,38 +64,42 @@ Federated shares are, in terms of data migration, like special external mounts,
 
 The following limitations impact the migration process:
 
-* While ownCloud fully respects letter casing for file and folder names, Kiteworks does not distinguish casings. If case conflicts happen during the migration process, a migration log file describing rclone responses and casing conflicts for files or directories is created. The ownCloud admin must resolve the conflicts to finalize the migration. For details see the xref:migration[Migration] description.
+* The ownCloud migration app is only functional on `amd64` platforms.
+
+* While ownCloud fully respects letter casing for file and folder names, Kiteworks does not distinguish casing. If case conflicts happen during the migration process, a migration log file describing rclone responses and casing conflicts for files or directories is created. The ownCloud admin must resolve the conflicts to finalize the migration. For details see the xref:migration[Migration] description.
 
 * When group shares have been defined, groups will not get created in Kiteworks. Instead, each member of the group will get an individual user share to the object shared.
 
-* Users have been able to login to ownCloud using either their display name, login name or email address. Kiteworks only allows login using the email address. The presence of the users email address in ownCloud is therefore a mandatory requirement.
+* In ownCloud, users can login either using their display name, login name or email address. Kiteworks only allows login using the email address. The presence of the users email address in ownCloud, which must be unique, is therefore a mandatory requirement.
 
-* Expiry dates for shares are not supported.
+* Expiry dates for shares are not migrated.
```

Comment:
This statement can be miss understood.

Which applies to public links only which are not supported in kiteworks. 
Expiry dates on shared resource in kiteworks is possible but not supported by owncloud.

Reply:
fixed